migrator/AppContext.swift

```diff
@@ -105,11 +108,16 @@ struct AppContext {
     static var duplicateFilesHandlingPolice: DuplicateFilesHandlingPolicy {
         return DuplicateFilesHandlingPolicy(rawValue: UserDefaults.standard.string(forKey: Self.duplicateFilesHandlingPolicyKey) ?? "overwrite") ?? .overwrite
     }
-    
+    static var urlExclusionsList: [URL?] {
```

Comment:
The previous name was `urlExclusionList` if you want to rename it `urlExclusionsList` you must change the reference to this variable in the other files of the project.

Reply:
Sorry @SMartorelli, this has been resolved with 76e635f3aabe016f27f36e4bbd13482c031cb380.

Reply:
Make that 2e308f95ff59e4f7322fbaee45aa842c200ae99d due to signing commit.